week04_approx_rl/homework_pytorch_main.ipynb

```diff
@@ -887,11 +887,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "from tqdm import trange\n",
```

Comment:
Just one more thing: as far as I see, there's already an import of `trange` above, so this one is redundant.